lib/tram/policy/error.rb

```diff
@@ -27,16 +27,20 @@ def self.new(value, **opts)
     # @return [Hash<Symbol, Object>] error tags
     attr_reader :tags
 
-    # The list of arguments for [I18n.t]
+    # @!attribute [r] scope
+    # @return [Array<String>] scope for error message translation
+    attr_reader :scope
+
+    # List of arguments for [I18n.t]
     #
     # @return [Array]
     #
     def item
-      [key, tags]
+      [key, tags.merge(scope: scope)]
```

Comment:
```suggestion
      [key, {scope: scope}.merge(tags)]
```

Reply:
Otherwise, if we use `errors.merge` scope of merged errors would be incorrect)

Reply:
@sclinede I rewrote the way we store scopes of the error and dropped the separate argument for scopes. Now they are just a part of the `tags`, so this merging is not necessary.